bftengine/src/bftengine/ClientsManager.cpp

```diff
@@ -358,8 +385,14 @@ std::unique_ptr<ClientReplyMsg> ClientsManager::allocateNewReplyMsgAndWriteToSto
 std::unique_ptr<ClientReplyMsg> ClientsManager::allocateReplyFromSavedOne(NodeIdType clientId,
                                                                           ReqId requestSeqNum,
                                                                           uint16_t currentPrimaryId) {
-  const uint32_t firstPageId = getReplyFirstPageId(clientId);
-  LOG_DEBUG(CL_MNGR, KVLOG(clientId, requestSeqNum, firstPageId));
+  if (!clientsInfo_[clientId].repliesInfo->findSafe(requestSeqNum)) {
+    LOG_FATAL(CL_MNGR, "Reply is not found for specified request" << KVLOG(clientId, requestSeqNum));
```

Comment:
Not sure whether it should be a fatal error.

Reply:
Changed to warning, thanks.

Reply:
Used LOG_DEBUG instead